src/main/java/org/apache/sysml/parser/dml/DmlSyntacticValidator.java

```diff
@@ -507,6 +507,19 @@ private void castAsScalarDeprecationCheck(String functionName, ParserRuleContext
 		}
 	}
 
+	private void convertNamespace(ArrayList<ParameterExpression> params) {
+		for (ParameterExpression p : params) {
```

Comment:
Doing that for all parameters of all builtin functions seems dangerous as it has the potential to break user-provided string parameters as well. Please restrict these modifications to the first parameter of eval calls.